core/domain/story_services.py

```diff
@@ -559,6 +559,28 @@ def get_completed_node_ids(user_id, story_id):
     return progress_model.completed_node_ids if progress_model else []
 
 
+def get_latest_completed_node_ids(user_id, story_id):
+    """Returns the  ids of the nodes lastest completed in the story.
```

Comment:
Nit: drop extra space before ids.

Also I am confused -- "lastest" isn't a word. Do you mean "last"? Or "latest"? If so -- do you mean "completed most recently" or "nodes that appear later in the story"?

Please rewrite to clarify. Ditto for Returns docstring below.

Reply:
Done.